massa-api/src/public.rs

```diff
@@ -544,9 +546,9 @@ impl Endpoints for API<Public> {
                 Map::with_capacity_and_hasher(addresses.len(), BuildMap::default());
             let mut endorsements: Map<Address, Set<EndorsementId>> =
                 Map::with_capacity_and_hasher(addresses.len(), BuildMap::default());
-            let mut final_sce_ledger_info: Map<Address, SCELedgerInfo> =
+            let mut final_balances: Map<Address, Amount> =
                 Map::with_capacity_and_hasher(addresses.len(), BuildMap::default());
-            let mut candidate_sce_ledger_info: Map<Address, SCELedgerInfo> =
+            let mut candidates_balances: Map<Address, Amount> =
```

Comment:
We need to document every API change in the API doc, and also ping Evgeni and Adrien because it will break all the tooling (eg. explorer etc...)

Reply:
I implemented the precedent behaviour with the new ledger so that it stays compatible.